### PR TITLE
Add the home page "call to action" / signup pattern

### PR DESCRIPTION
**Pull request by carolinan**
Adding this pattern here for now, and we can discuss if it should be in the pattern directory or the theme.

I am concerned the text might indicate to some users that the theme has some sort of newsletter functionality.
Can we come up with something more neutral?

### CODE REVIEW

**Comment by beafialho** [comment]
> I am concerned the text might indicate to some users that the theme has some sort of newsletter functionality.
> Can we come up with something more neutral?

Good point, how about this?

Captura de ecrã 2022-08-17, às 10 44 02

I noticed the line height seems a little too large to me, compared to the Figma specs:

Captura de ecrã 2022-08-17, às 10 47 52

**Comment by carolinan** [comment]
When I updated the text, the question mark was on a third line. So I removed the 40% width of the left column.
That meant that the line did not extend as close to the text as in the design:

Screen Shot 2022-08-17 at 12 10 52

We should probably test with a row block too instead of a columns block.

**Comment by carolinan** [comment]
I tested a row with a stack with the paragraph and button on the left, and the separator on the right, and it did not work, the separator is invisible.

row block with nested blocks

**Comment by beafialho** [comment]
> When I updated the text, the question mark was on a third line. So I removed the 40% width of the left column.
> That meant that the line did not extend as close to the text as in the design:

Thank you, I think that's ok.

File: patterns/call-to-action.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * Keywords: Call to action, signup, newsletter
+ * Block Types: core/buttons
+ */
+?>
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide">
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.2"}},"fontSize":"x-large"} -->
+		<p class="has-x-large-font-size" style="line-height:1.2">Got any book recommendations?</p>
```
Comment (mikachan) [inline patterns/call-to-action.php]: Does this content need to be escaped/translated?

Comment (carolinan) [reply patterns/call-to-action.php]: Yes!

**Review by mikachan — APPROVED** [review]
🚢🚢

**Pull request merged**